GiEnJul/ClientApp/src/routes/registerGiver/LocationGiver.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Button, ButtonToolbar } from 'reactstrap';
+import { Route } from 'react-router-dom'
+import Dropdown from 'react-dropdown';
+import 'react-dropdown/style.css';
+
+type Props = {
+  nextStep: () => void,
+  prevStep?: () => void,
+  handleLocationChange: (newLocation: Object) => void,
+  values: { location: string | undefined; name: string | undefined; email: string | undefined; tlf: number | undefined; familiyType: string | undefined; }
```

Comment:
These could probably be shortened to e.g. `location?: string` instead of `location: string | undefined`. I think the result would be the same, but it would look a bit cleaner.

Reply:
Thanks! Made this a lot cleaner :D